_docs/documenting_rest_api_endpoints/docapis_doc_code_samples.md

```diff
@@ -226,7 +226,7 @@ var settings = {
   "async": true,
   "crossDomain": true,
   "dataType": "json",
-  "url": "https://simple-weather.p.mashape.com/surfreport/25",
+  "url": "https://simple-weather.p.mashape.com/surfreport/25&days=1",
```

Comment:
In the prose that appears after the code sample, you write "The endpoint limits the days returned to 1 in order to increase the download speed."   `surfreport` returns 3 days by default, so I restricted the query to return 1 day to match the prose.